src/libs/actions/Report.js

```diff
@@ -672,6 +672,29 @@ function fetchActions(reportID) {
         });
 }
 
+/**
+ * Get the actions of a report during pagination
+ *
+ * @param {Number} reportID
+ * @param {Number} [offset]
+ */
+function fetchActionsWithPagination(reportID, offset) {
+    Onyx.set(ONYXKEYS.IS_LOADING_MORE_REPORT_ACTIONS, true);
+    fetchActions(reportID, offset)
+        .finally(() => Onyx.set(ONYXKEYS.IS_LOADING_MORE_REPORT_ACTIONS, false));
```

Comment:
Shouldn't this be corrected as well? is this key exists `ONYXKEYS.IS_LOADING_MORE_REPORT_ACTIONS`?

Reply:
I had deleted this function as its not used anywhere now. Looks like it came back while resolving the signed commit issues. Removed it.